package.json

```diff
@@ -22,7 +22,7 @@
     "prepare": "ts-patch install && npm run build"
   },
   "engines": {
-    "node": ">=16.0.0",
+    "node": ">=12.18.3",
```

Comment:
Currently. `yorkie-js-sdk` node engine version is over 16, but we are having conflicts when installing some modules.  So I changed to required node version
https://nodejs.org/en/about/releases/
<img width="854" alt="Screen Shot 2022-01-30 at 12 02 13 AM" src="https://user-images.githubusercontent.com/4694139/151665923-bd363017-6a8b-483e-ae55-ce4004eb34f3.png">

```
$ npm i @microsoft/api-extractor --save-dev
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: yorkie-js-sdk@0.2.0
npm ERR! notsup Not compatible with your version of node/npm: yorkie-js-sdk@0.2.0
npm ERR! notsup Required: {"node":">=16.0.0","npm":">=7.1.0"}
npm ERR! notsup Actual:   {"npm":"8.3.2","node":"v12.18.3"}
```